mlflow/entities/model_registry/registered_model_detailed.py

```diff
@@ -1,15 +1,21 @@
 from mlflow.entities.model_registry.model_version_detailed import ModelVersionDetailed
 from mlflow.entities.model_registry.registered_model import RegisteredModel
 from mlflow.protos.model_registry_pb2 import RegisteredModelDetailed as ProtoRegisteredModelDetailed
+from mlflow.utils import experimental
 
 
+@experimental
 class RegisteredModelDetailed(RegisteredModel):
-    # __init__ method to initialize fields
+    """
+    MLflow entity for Registered Model Detailed.
+    Provides additional metadata data for registered model in addition to information in
+    :py:class:`mlflow.entities.model_registry.RegisteredModel`.
+    """
+
     def __init__(self, name, creation_timestamp, last_updated_timestamp=None, description=None,
                  latest_versions=None):
         # Constructor is called only from within the system by various backend stores.
         super(RegisteredModelDetailed, self).__init__(name)
-        self._name = name
```

Comment:
This `_name` was redundant since this is already defined in base class `RegisteredModel`. Removing it.